specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/webapplicationfirewall.json

```diff
@@ -686,6 +690,30 @@
           "description": "When matchVariable is a collection, operator used to specify which elements in the collection this exclusion applies to."
         }
       }
+    },
+    "BotDetection": {
+      "description": "Allows the detection of bots with a customizable list of exclusions",
+      "properties": {
+        "state": {
+          "description": "Describes if the bot detection is in enabled or disabled state.",
```

Comment:
"Describes if bot-detection is enabled or disabled" is this better?

Reply:
I prefer the current description, not because it sounds better, but because it's consistent with the rest of the enabled/disabled properties